docs/src/api/class-clock.md

```diff
@@ -42,6 +43,14 @@ Tells `@sinonjs/fake-timers` to increment mocked time automatically based on the
 Relevant only when using with [`option: shouldAdvanceTime`]. Increment mocked time by advanceTimeDelta ms every advanceTimeDelta ms change
 in the real system time (default: 20).
 
+
+## async method: Clock.next
+* since: v1.45
+- returns: <[int]> Fake milliseconds since the unix epoch.
+
+Advances the clock to the the moment of the first scheduled timer, firing it.
```

Comment:
What happens if there are no timers scheduled?

Reply:
I'll need to follow up to respond.